tests/test_unary_pointwise_ops.py

```diff
@@ -457,3 +457,17 @@ def test_accuracy_repeat(shape, sizes, dtype):
         res_out = inp.repeat(*sizes)
 
     gems_assert_close(res_out, ref_out, dtype)
+
+
+@pytest.mark.logical_not
+@pytest.mark.parametrize("shape", POINTWISE_SHAPES)
+@pytest.mark.parametrize("dtype", ALL_FLOAT_DTYPES)
+def test_accuracy_logical_not(shape, dtype):
```

Comment:
Add tests for BOOL_TYPES and INT_DTYPES.

Reply:
Fixed, thx